qutebrowser/config/configdata.yml

```diff
@@ -385,6 +385,25 @@ qt.workarounds.locale:
     However, It is expected that distributions shipping QtWebEngine 5.15.3
     follow up with a proper fix soon, so it is disabled by default.
 
+qt.workarounds.disable_accelerated_2d_canvas:
+  type:
+    name: String
+    valid_values:
+      - always: Disable accelerated 2d canvas
+      - auto: Disable on Qt6 < 6.6.0, enable otherwise
+      - never: Enable accelerated 2d canvas
```

Comment:
While we have the always/auto/never convention elsewhere, we could pick better names here to avoid the double negative.

An obvious choice would be `enable` / `auto` / `disable`, but that's also ambiguous because it's not clear if we enable accelerated 2d canvas, or if we enable the workaround.

I suppose it's also okay to keep it as is.